### PR TITLE
Fixes AB#49

### PR DESCRIPTION
**Pull request by peter-murray**
Hi,

This pull request fixes issue(s) [AB#49](https://dev.azure.com/octoorg/f9fecbbc-d2ce-41aa-b745-de61141f55a7/_workitems/edit/49).

A summary of changes are:
- Adding star ratings for books
 
Please proceed with a review as soon as all the status checks are valid.

### CODE REVIEW

File: src/main/java/com/github/demo/model/Book.java [hunk 1]
```diff
@@ -30,6 +33,14 @@ public Book(String author, String title, String cover) {
         this.cover = cover;
     }
 
+    public Book(String author, String title, String cover, int rating) {
+        this();
```
Comment (pmanlukas) [inline src/main/java/com/github/demo/model/Book.java:37]: Great work on your first day!

File: src/main/webapp/books.html [hunk 2]
```diff
@@ -80,6 +80,13 @@
                   <h4 class="media-heading"><a href="#">[[${book.title}]]</a></h4>
                   <h5 class="media-heading"> by <a href="#">[[${book.author}]]</a></h5>
                   <span>Status: </span><span class="text-success"><strong>In Stock</strong></span>
+                  <!-- Rating shown in stars -->
+                 <div>
```
Comment (pmanlukas) [inline src/main/webapp/books.html:84]: Like the changes to the design

**Review by pmanlukas — APPROVED** [review]
LGTM

**Comment by peter-murray** [comment]
The feature branch deployment looks good @pmanlukas

**Comment by peter-murray** [comment]
@pmanlukas Can you please merge?

**Pull request merged**